database/migrate.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const postgrator = require('postgrator')
+const path = require('path')
+const minimist = require('minimist')
+const config = require('./../src/lib/config')
+
+postgrator.setConfig({
+  migrationDirectory: path.join(__dirname, '/migrations'),
+  schemaTable: 'schemaversion', // optional. default is 'schemaversion'
+  driver: 'pg', // or mysql, mssql
+  host: config.get('pgdb.host', '127.0.0.1'),
```

Comment:
Could you please add env variables as the main config option? Something like: ` port: process.env.SERVICE_PORT || config.get(...)`. Will probably needed for production and make the life easier on CI.